frontend/src/main/scala/bloop/exec/Forker.scala

```diff
@@ -57,37 +57,64 @@ final case class Forker(javaEnv: JavaEnv, classpath: Array[AbsolutePath]) {
       opts: CommonOptions,
       extraClasspath: Array[AbsolutePath] = Array.empty
   ): Task[Int] = {
-    import scala.collection.JavaConverters.{propertiesAsScalaMap, mapAsJavaMapConverter}
     val fullClasspath = (classpath ++ extraClasspath).map(_.syntax).mkString(pathSeparator)
     val java = javaEnv.javaHome.resolve("bin").resolve("java")
     val classpathOption = "-cp" :: fullClasspath :: Nil
     val appOptions = mainClass :: args.toList
     val cmd = java.syntax :: javaEnv.javaOptions.toList ::: classpathOption ::: appOptions
+    val logTask =
+      if (logger.isVerbose) {
+        val debugOptions =
+          s"""
+             |Fork options:
+             |   command      = '${cmd.mkString(" ")}'
+             |   cwd          = '$cwd'
+             |   classpath    = '$fullClasspath'
+             |   java_home    = '${javaEnv.javaHome}'
+             |   java_options = '${javaEnv.javaOptions.mkString(" ")}""".stripMargin
+        Task(logger.debug(debugOptions))
+      } else Task.unit
+    logTask.flatMap(_ => Forker.run(cwd, cmd, logger, opts))
+  }
+
+}
+
+object Forker {
+
+  /** The code returned after a successful execution. */
+  final val EXIT_OK = 0
```

Comment:
Make both of them `private[Forker]`?

Reply:
That value is being used in other places.